radicle-cli/src/commands/ls.rs

```diff
@@ -53,10 +53,10 @@ pub fn run(_options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
         let Ok(Project { name, description, .. }) = repo.project_of(profile.id()) else { return };
         let head = term::format::oid(head);
         table.push([
-            term::format::bold(name),
-            term::format::tertiary(id),
-            term::format::secondary(head),
-            term::format::italic(description),
+            &term::format::bold(name),
+            &term::format::tertiary(id),
+            &term::format::secondary(head),
+            &term::format::italic(description),
         ]);
```

Comment:
I don't think the `&` are needed anymore
```suggestion
            term::format::bold(name),
            term::format::tertiary(id),
            term::format::secondary(head),
            term::format::italic(description),
        ]);
```